config/routes.rb

```diff
@@ -52,6 +52,10 @@ def show_step(name)
         edit_step :contact_details
       end
 
+      namespace :cases do
```

Comment:
It has no real impact but I find it useful to declare the routes in the same logical order that they appear in the journey.
So perhaps we can move this namespace below the `address` namespace?